applications/nrf_desktop/description.rst

```diff
@@ -233,13 +235,14 @@ In this state, the nRF Desktop mouse forwards the data from the motion sensor to
 #. When the HID input report is sent to the host, ``hid_report_sent_event`` is submitted.
    The motion sensor sample is triggered and the sequence repeats.
 
-If the device is connected through Bluetooth, the :ref:`nrf_desktop_hid_state` uses a pipeline that consists of two HID reports that it creates upon receiving the first ``motion_event``.
+If the device is connected through Bluetooth LE or :ref:`nrf_desktop_usb_state_sof_synchronization` is enabled, the :ref:`nrf_desktop_hid_state` uses a pipeline that consists of two HID reports that it creates upon receiving the first ``motion_event``.
```

Comment:
shouldn't we mention that this option is turned on and we are connected by USB?

Reply:
I assumed that it's obvious :). Anyway, added